lib/actions/ui.js

```diff
@@ -453,9 +453,7 @@ export function setLocale(locale) {
       dispatch(updateLocale({ locale: matchedLocale, messages }))
 
       // Update the lang attribute in the root <html> element.
-      // (The lang is the first portion of the locale.)
-      const lang = matchedLocale.split('-')[0]
-      document.documentElement.lang = lang
+      document.documentElement.lang = matchedLocale
```

Comment:
The HTML `lang` tag can be a locale (language-suffix format) so we should be good here.